src/app/organizations/organization-details/organization-details.component.scss

```diff
@@ -90,35 +82,37 @@
 
 .manage__members {
   .col-lg-7,
-  .col-lg-5,
-  .col-lg-2 {
+  .col-lg-6,
+  .col-lg-1,
+  .col-xl-5,
+  .col-xl-2 {
     padding: 0;
   }
```

Comment:
is there a better way of doing it? just asking.

Reply:
There is but I thought of this only later.
I added p-0 bootstrap class to 2 elements that had these classes and we have cleaner CSS now.